google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -274,6 +284,35 @@ public void testBatchWriteWithCommittedStreamEU()
     assertEquals(3, response2.get().getAppendResult().getOffset().getValue());
   }
 
+  @Test
+  public void testProto3OptionalBatchWriteWithCommittedStream()
+      throws IOException, InterruptedException, ExecutionException {
+    WriteStream writeStream =
+        client.createWriteStream(
+            CreateWriteStreamRequest.newBuilder()
+                .setParent(tableId)
+                .setWriteStream(
+                    WriteStream.newBuilder().setType(WriteStream.Type.COMMITTED).build())
+                .build());
+    StreamWriter streamWriter =
+        StreamWriter.newBuilder(writeStream.getName())
+            .setWriterSchema(ProtoSchemaConverter.convert(FooOptionalType.getDescriptor()))
+            .build();
+    LOG.info("Sending one message");
+
+    ApiFuture<AppendRowsResponse> response =
+        streamWriter.append(CreateProtoOptionalRows(new String[] {"aaa"}), 0);
+    assertEquals(0, response.get().getAppendResult().getOffset().getValue());
+
+    LOG.info("Sending two more messages");
+    ApiFuture<AppendRowsResponse> response1 =
```

Comment:
Try a null value?

Reply:
I have changed the last request to pass an empty string, which matches the default value.